schema/tracer_provider.yaml

```diff
@@ -22,6 +22,38 @@ properties:
 required:
   - processors
 $defs:
+  SpanKind:
+    type:
+      - string
+      - "null"
+    enum:
+      - INTERNAL
+      - SERVER
+      - CLIENT
+      - PRODUCER
+      - CONSUMER
+    enumDescriptions:
+      INTERNAL: INTERNAL, an internal span.
+      SERVER: SERVER, a server span.
+      CLIENT: CLIENT, a client span.
+      PRODUCER: PRODUCER, a producer span.
+      CONSUMER: CONSUMER, a consumer span.
+  SpanParent:
```

Comment:
#427 made me realize perhaps the rules need to capture the existing knobs of the `ParentBasedSampler` and went with this approach. @PeterF778 let me know if this makes sense

Reply:
Thanks, definitely should have been `ExperimentalSpanParent`